### PR TITLE
auto deploy for meshery operator

### PR DESCRIPTION
**Pull request by kumarabd**
Signed-off-by: kumarabd <abishekkumar92@gmail.com>

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: ui/components/MeshConfigComponent.js [hunk 1]
```diff
@@ -362,6 +362,7 @@ class MeshConfigComponent extends React.Component {
     }, (result) => {
       this.props.updateProgress({ showProgress: false });
       if (typeof result !== 'undefined') {
+        // Prompt
```
Comment (kumarabd) [inline ui/components/MeshConfigComponent.js:365]: @vineethvanga18 Prompt component to be used here

File: ui/components/MeshConfigComponent.js [hunk 2]
```diff
@@ -465,6 +466,7 @@ class MeshConfigComponent extends React.Component {
...
```
Comment (kumarabd) [inline ui/components/MeshConfigComponent.js:469]: @vineethvanga18 And here

File: internal/graphql/resolver/operator.go [hunk 3]
```diff
@@ -40,7 +41,20 @@ func (r *Resolver) changeOperatorStatus(ctx context.Context, status model.Status
...
 	go func(del bool, kubeclient *mesherykube.Client) {
-		err := initialize(kubeclient, del)
+		err := r.cleanEntries(del)
+		if err != nil {
+			r.Log.Error(err)
+			r.operatorChannel <- &model.OperatorStatus{
+				Status: status,
+				Error: &model.Error{
+					Code:        "",
+					Description: err.Error(),
```
Comment (leecalcote) [inline internal/graphql/resolver/operator.go:51]: @kumarabd should this be filled-in?

File: internal/graphql/resolver/operator.go [hunk 4]
```diff
@@ -279,3 +300,23 @@ func applyYaml(client *mesherykube.Client, delete bool, file string) error {
 
 	return nil
 }
+
+func (r *Resolver) cleanEntries(del bool) error {
+	if del {
+		r.Log.Info("Removing Operator")
+		objs, err := getOperator(r.DBHandler)
+		if err != nil {
```
Comment (leecalcote) [inline internal/graphql/resolver/operator.go:308]: @kumarabd is the Operator orphaned at this point? What should users do about this? Can you provide an probable cause and suggested remediation?

File: ui/components/MeshConfigComponent.js (hunk 1 above)
Comment (leecalcote) [reply ui/components/MeshConfigComponent.js:365]: @vineethvanga18 @kumarabd does this PR need updates from Vineeth before merging?

File: internal/graphql/resolver/operator.go (hunk 3 above)
Comment (kumarabd) [reply internal/graphql/resolver/operator.go:51]: We don't have the code at the moment. So no...

File: ui/components/MeshConfigComponent.js (hunk 1 above)
Comment (kumarabd) [reply ui/components/MeshConfigComponent.js]: yes

**Comment by kumarabd** [comment]
@vineethvanga18  The prompt component doesn't seem to work. Will you check this?

**Comment by leecalcote** [comment]
Was this issue fixed in this PR?
Screen Shot 2021-03-10 at 12 17 55 PM

**Comment by leecalcote** [comment]
Will the component versions show based on changes in this PR?
Screen Shot 2021-03-10 at 12 12 39 PM

**Comment by kumarabd** [comment]
> Was this issue fixed in this PR?
> Screen Shot 2021-03-10 at 12 17 55 PM

Yes this is fixed in this PR.

**Comment by kumarabd** [comment]
> Will the component versions show based on changes in this PR?
> Screen Shot 2021-03-10 at 12 12 39 PM

Yes this is in the upcoming PR.

**Pull request merged**